### PR TITLE
Remove unused dependencies

### PR DESCRIPTION
**Pull request by amstilp**
Remove packages that are not actually being used. This will cut down on dependabot noise.

### CODE REVIEW

**Comment by amstilp** [comment]
@jmcarson when you get a chance, can you also check that these dependencies are not actually needed?

**Review by jmcarson — APPROVED** [review]
looks good, thanks.

File: requirements/dev-requirements.in [hunk 1]
```diff
@@ -1,10 +1,10 @@
 -c requirements.txt
 -c test-requirements.txt
 
-Werkzeug # https://github.com/pallets/werkzeug
```
[the remark this answers is not in the document]
Comment (amstilp) [reply requirements/dev-requirements.in:4]: Sounds good - I added it back to the dev requirements.

**Comment by amstilp** [comment]
Looks like removing argon2-cffi broke admin logins. oops. I'll add it back in in a new pull request.

**Pull request merged**